server/src/test/java/com/broadcom/lsp/cobol/usecases/TestParagraphNotDefined.java

```diff
@@ -0,0 +1,41 @@
+package com.broadcom.lsp.cobol.usecases;
+
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TestParagraphNotDefined extends NegativeUseCase {
+    private static final String TEXT =
+            "        IDENTIFICATION DIVISION.\r\n"
+                    + "        PROGRAM-ID. test1.\r\n"
+                    + "        DATA DIVISION.\r\n"
+                    + "        WORKING-STORAGE SECTION.\r\n"
+                    + "        PROCEDURE DIVISION.\r\n"
+                    + "        PROGA.\r\n"
+                    + "           PERFORM PRINT-DATA.";
```

Comment:
`PROGA` is a paragraph definition, you need to wrap it also
```suggestion
                    + "        {#*PROGA}.\r\n"
                    + "           PERFORM {#PRINT-DATA}.";
```

Reply:
Done